python/rmm/_lib/lib.pxd

```diff
@@ -64,8 +64,9 @@ cdef extern from "rmm/rmm.h" nogil:
         CudaDefaultAllocation = 0,
         PoolAllocation = 1,
         CudaManagedMemory = 2,
+        ManagedPoolAllocation = 3,
```

Comment:
Please don't add this option, it already exists. Note that the existing options are all powers of two so they  are composable. Specify a managed pool by using `PoolAllocation | CudaManagedMemory`.  I don't want to double the number of options every time we add a new orthogonal feature -- the next one would make this go from 4 to 8 options.

Reply:
@harrism I added it per a request from @jrhemstad, but happy to switch it back.